src/Components/UI/Forms/ProductForm/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback } from "react";
+import { useFormik, FormikValues } from "formik";
+import { IProduct } from "../../../../models";
+import { InputText } from "primereact/inputtext";
+import styled from "styled-components";
+import { Button } from "primereact/button";
+import uuid from "react-uuid";
+import { QueryClient, useMutation } from "react-query";
+import { apiUrl, URL } from "../../../../shared";
+import { ProductService } from "../../../../shared/services/product-service";
+import { Link } from "react-router-dom";
+
+const Root = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 30px;
+`;
+
+const createProduct = (product: Partial<IProduct>) =>
+  ProductService.addProduct(product as IProduct);
+
+export const ProductForm = () => {
+  const { mutate } = useMutation(createProduct, {});
+
+  const handleFormSubmit = useCallback((data: Partial<IProduct>) => {
+    console.log(data, "dsd");
+    mutate(data);
```

Comment:
Please create `ICreateProduct` interface (under product models) and prepare the Create Product 
```suggestion
    type ICreateProduct = Omit<Product, "id">;

   
    const newPolicy: ICreateProudct = {
       name: values.name,
       brand: values.brand,
       image: values.image,
       price:  values.price || 0
    }
    mutate(newPolicy);
```